examples/persons/package.json

```diff
@@ -19,17 +19,17 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@apollo/react-hooks": "^3.0.0",
-    "react": "^16.8.1",
-    "react-dom": "^16.8.1",
+    "@apollo/react-hooks": "^3.1.1",
+    "react": "^16.10.1",
+    "react-dom": "^16.10.1",
     "reason-apollo": "^0.17.0",
     "reason-apollo-hooks": "../../",
     "reason-react": ">=0.7.0"
   },
   "devDependencies": {
-    "bs-platform": "^5.0.6",
+    "bs-platform": "^5.2.0",
     "css-loader": "^3.2.0",
-    "graphql_ppx": "^0.2.8",
+    "@baransu/graphql_ppx_re": "^0.2.0-beta.2",
```

Comment:
very good!

Reply:
Yeah so cool to see activity again in this package. It has so much potential.